src/msw/dirdlg.cpp

```diff
@@ -71,15 +71,11 @@ struct IShellItem : public IUnknown
     virtual HRESULT wxSTDCALL Compare(IShellItem*, DWORD, int*) = 0;
 };
 
-#endif // #ifndef __IShellItem_INTERFACE_DEFINED__
-
-// Define this GUID in any case, even when __IShellItem_INTERFACE_DEFINED__ is
-// defined in the headers we might still not have it in the actual uuid.lib,
-// this happens with at least VC7 used with its original (i.e. not updated) SDK
-// and there is no harm in defining the GUID unconditionally.
```

Comment:
Hmm, so it looks like this would break VC7 build? Not that I care much about it, and we should probably just officially drop support for it, but as long as we keep it, we probably shouldn't do this...

Reply:
Somehow I assumed it wasn't supported in wx3.2. I'll revert it and add a `__VISUALC__` guard.